src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,60 @@
+const api = 'https://api.spacexdata.com/v3/rockets';
+
+const GET_ROCKETS = 'GET_ROCKETS';
+const RESERVE_ROCKET = 'RESERVE_ROCKET';
+const UNRESERVE_ROCKET = 'UNRESERVE_ROCKET';
+
+// Action Creators
+
+export const getRockets = () => async (dispatch) => {
+  await fetch(api)
+    .then((rockets) => rockets.json())
+    .then(
+      (data) => dispatch({ type: GET_ROCKETS, payload: data }),
```

Comment:
Nice work fetching rockets!.
But it's required and also it's a good practice to use DUCK Pattern[ learn more](https://github.com/erikras/ducks-modular-redux)
But in brief, action creators should be in a separate function like that:
`getRocketsAction = (rockets) => return { type:GET_ROCKETS, payLoad:rockets }`
etc..
Also, it's a good practice to put the fetching logic in a separate module (OPTIONAL BUT RECOMMENDED)

Reply:
Here I respected the DUCK pattern
I just called the API in the action creator to have the code clearly and with good efficiency.

Also thank you so much!

Reply:
just like the example i wrote, anyways you can merge